samples/Kconfig

```diff
@@ -16,14 +16,58 @@ config NCS_SAMPLES_DEFAULTS
 	help
 	  Use the default configuration for NCS samples.
 
-config NCS_SAMPLE_EMPTY_APP_CORE_CHILD_IMAGE
-	bool "Enable empty_app_core as a child image"
-	default n
+if SOC_SERIES_NRF53X
+
+config NCS_CHILD_IMAGE_SAMPLE_EMPTY_APP_CORE
```

Comment:
Why are you changing the Kconfig setting name ?

This causes noise for customers using the old name in their conf files when the upmerge to a newer NCS release.
We should therefore not rename settings unless there is a very good reason to do so.

Reply:
okay, I just wanted to be consistent with the other configs. I thought it was cleaner to have NCS_CHILD_IMAGE_SAMPLE_xx instead of NCS_SAMPLE_xx_CHILD_IMAGE. I do agree it creates noise (see the failing ble CI), and don't have a strong opinion so I'll revert and make the others look the same.